tests/compile.sh

```diff
@@ -24,16 +24,17 @@ fi
 readonly ARGC=$#
 
 if [[ $ARGC -lt 2 ]]; then
-  echo "Usage: $0 MACHINE_ID [ MAKE_OPT [ BUILD_NR ] [ clean_before ] [ clean_after ] ]"
+  echo "Usage: $0 MACHINE_ID [ MAKE_OPT [ BUILD_NR ] [ COMPILE_NR ] [ RT_COMPILER ] [ clean_before ] [ clean_after ] ]"
   echo Valid MACHINE_IDs:
   echo $( ls -1 ../cmake/configure_* | sed s:.*configure_::g | sed s:\.cmake:: ) | fold -sw72
   exit 1
 else
   MACHINE_ID=$1
   MAKE_OPT=${2:-}
   COMPILE_NR=${3:+_$3}
-  clean_before=${4:-YES}
-  clean_after=${5:-YES}
+  RT_COMPILER=${4:-intel}
```

Comment:
RT_COMPILER is ${5:-YES}? Please see line 27.

Reply:
Thanks! Addressed with https://github.com/ufs-community/ufs-weather-model/pull/1718/commits/e7e3ec23cb1619450c4e361cf099787bceac5166